src/api/api.js

```diff
@@ -5,67 +5,15 @@ export function getInstitutions() {
   return new Promise(resolve => {
     resolve({
       institutions: [
-        { lei: 'B90YWS6AFX2LGWOXJ1LD', name: 'Bank 0' },
-        { lei: 'BANK1LEIFORTEST12345', name: 'Bank 1' }
+        { lei: '01KWVG908KE7RKPTNP46', name: 'Bank 0' },
```

Comment:
You can swap these for a new institution to test out the logic locally. Let me know if you need another `lei` to test.